### PR TITLE
darkarmour dep osslsigncode fix #3032

### LINKED ISSUE #3032: darkarmour
darkarmour
https://github.com/bats3c/darkarmour

### PR DESCRIPTION
**Pull request by dualfade**
darkarmour dependency --
please review --

### CODE REVIEW

File: packages/osslsigncode/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=osslsigncode
+pkgver=242.c0d9569
+pkgrel=1
+pkgdesc='A small tool that implements part of the functionality of the Microsoft tool signtool.exe.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-microsoft')
```
Comment (Edu4rdSHL) [inline packages/osslsigncode/PKGBUILD:9]: `blackarch-microsoft` is not a valid group. I think that we should put it in `blackarch-windows` and `blackarch-automation`

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Hello, please fix.

File: packages/osslsigncode/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/osslsigncode/PKGBUILD:9]: Duh hah. fixing.

**Review by Edu4rdSHL — APPROVED** [review]

**Comment by Edu4rdSHL** [comment]
Thank you a lot, @dualfade !

**Pull request merged**